### ISSUE TITLE
REPL exit message UNIX specific and doesn't work on Windows

### ISSUE THREAD

**Issue by stmuk** [rakudo/rakudo#2186]
````
To exit type 'exit' or '^D'
> ^D
===SORRY!=== Error while compiling:
Bogus statement
------> <BOL><HERE>
    expecting any of:
        prefix
        term
````

^C is needed on Windows

**Comment by zoffixznet** [comment]
Related:
https://github.com/rakudo/rakudo/issues/1969
https://github.com/rakudo/rakudo/issues/1677

**Comment by vrurg** [comment]
Closed via #3230
